testrunner/index.html~

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
I believe it means that it was created a temp file. Which process created it; who knows.